EatGoodNaija.Server/Migrations/20240416060657_w3.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EatGoodNaija.Server.Migrations
+{
+    public partial class w3 : Migration
```

Comment:
Nothing is added to this migration. What purpose is it serving in the application?